pkg/orchestration/wiring/internaldns/plugin.go

```diff
@@ -27,15 +27,6 @@ const (
 	kubeIngressRefExample                                   = "ingress-internal-01.ap-southeast-2.paas-dev1.kitt-inf.net"
 )
 
-type Alias struct {
```

Comment:
pulled out into their own file because that's the pattern I see with other providers.